.ci/build-mac-arm64.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh -ex
+
+brew update
+brew install -f --overwrite nasm ninja git p7zip create-dmg ccache pipenv llvm@16 sdl2 glew cmake faudio qt@6 ffmpeg@6 vulkan-headers
```

Comment:
Turning off FAudio resolves an issue with homebrew. Nas had to do the same thing on master.
```suggestion
brew install -f --overwrite nasm ninja git p7zip create-dmg ccache pipenv llvm@16 sdl2 glew cmake qt@6 ffmpeg@6 vulkan-headers
```